webpack.config.dev.js

```diff
@@ -0,0 +1,22 @@
+const merge = require('webpack-merge');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+let dev = {
```

Comment:
I'm sorry but I won't change it https://mathiasbynens.be/notes/es6-const I'm one of the guys who prefer to use that only for requires because we are sure that I'm importing something from other place that I wont change, on this case I can change this variables whatever I want. 
If this is not ok for you you don't have to accept it xD